homeassistant/components/device_tracker/__init__.py

```diff
@@ -182,12 +193,22 @@ def see(hass: HomeAssistantType, mac: str = None, dev_id: str = None,
                 setup = await hass.async_add_job(
```

Comment:
Do all these custom scanner/setup methods also go away in the final state (`async_get_scanner`, `get_scanner`, etc...)?  I would think all setup logic, regardless of how it's implemented would go in `setup_platform` or `setup_entry`.

Reply:
Yes, all of that will go away.